gantt-chart/opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "ganttChartDashboards",
-  "version": "2.4.1.0",
+  "version": "2.4.2.0",
   "opensearchDashboardsVersion": "2.4.0",
```

Comment:
`opensearchDashboardsVersion` also needs to be updated. @gaiksaya

Reply:
It wasn't 2.4.1 so I kept it as it is.

Reply:
Looks like all checks are successful as well.

Reply:
it should be `2.4.2` (i fixed it). it was `2.4.0` because PR https://github.com/opensearch-project/dashboards-visualizations/pull/141/files didn't bump it when increasing to `2.4.1.0`. Not sure where is the auto version bump